torch/_higher_order_ops/cond.py

```diff
@@ -241,6 +241,43 @@ def create_fw_bw_graph_branches(true_fn, false_fn, *operands):
         return fw_true_graph, fw_false_graph, joint_true_graph, joint_false_graph
 
 
+def create_bw_fn(
+    fn: Callable,
+    num_primals: int,
+    include_set: torch._C.DispatchKeySet,
+    exclude_set: torch._C.DispatchKeySet,
+):
+    from torch._functorch.aot_autograd import AOTConfig, create_joint
+    from torch._higher_order_ops.utils import prepare_fw_with_masks
+
+    dummy_aot_config = AOTConfig(
+        fw_compiler=None,  # type: ignore[arg-type]
+        bw_compiler=None,  # type: ignore[arg-type]
+        partition_fn=None,  # type: ignore[arg-type]
+        decompositions={},
+        num_params_buffers=0,
+        aot_id=0,
+        keep_inference_input_mutations=False,
+    )
+
+    def bw_fn(*grads_and_primals):
+        grads = grads_and_primals[:-num_primals]
+        primals = grads_and_primals[-num_primals:]
+        bw_f = create_joint(prepare_fw_with_masks(fn), aot_config=dummy_aot_config)
+        # TODO: somehow we still need to call torch.enable_grad() here, otherwise the grads
+        # info are gone.
```

Comment:
this is for if grad_mode is enabled, yes?
we could add an assertion here for that